src/analysis/retail/druid/restoration/CombatLogParser.ts

```diff
@@ -119,12 +120,15 @@ class CombatLogParser extends CoreCombatLogParser {
     rampantGrowth: RampantGrowth,
     overgrowth: Overgrowth,
     buddingLeaves: BuddingLeaves,
-    Dreamstate: Dreamstate,
+    dreamstate: Dreamstate,
 
     // Mana Tab
     manaTracker: ManaTracker,
     hpmDetails: RestoDruidHealingEfficiencyDetails,
     hpmTracker: HealingEfficiencyTracker,
+
+    //Tier
+    VotI2pc: VotI2pc,
```

Comment:
This should probably be `voti2pc: VotI2pc`:
```suggestion
    voti2pc: VotI2pc,
```